pcsx2/GS/Renderers/HW/GSTextureCache.cpp

```diff
@@ -2023,6 +2023,9 @@ GSTextureCache::Target* GSTextureCache::CreateTarget(GIFRegTEX0 TEX0, const GSVe
 		GL_CACHE("TC: Lookup %s(Color) %dx%d FBMSK %08x, miss (0x%x, TBW %d, %s) draw %d", is_frame ? "Frame" : "Target",
 			size.x, size.y, fbmask, TEX0.TBP0, TEX0.TBW, psm_str(TEX0.PSM), g_gs_renderer->s_n);
 	}
+	// Avoid making garbage targets (usually PCRTC).
+	if (GSVector4i::loadh(size).rempty())
```

Comment:
Maybe but it at the start of the function?

Reply:
That is the start of the function?

Reply:
I meant before the logs, or do we want them to paste?